test/puppetlabs/http/client/sync_ssl_test.clj

```diff
@@ -183,5 +183,5 @@
         (is (java-unsupported-protocol-exception?
               (java-https-get-with-protocols ["TLSv1.2"] nil))))
       (testing "clojure sync client"
-        (is (thrown? SSLException (clj-https-get-with-protocols ["TLSv1.2"] nil)))))))
+        (is (thrown? ConnectionClosedException (clj-https-get-with-protocols ["TLSv1.2"] nil)))))))
```

Comment:
oh fun, looks like Java 11 throws `SSLHandshakeException`

Reply:
Huh, it didn't locally for me.  Yay for Java flavor nuances :(

Reply:
Appears to have changed between 11.0.2 and 11.0.13.